src/components/common/volunteers-list.module.scss

```diff
@@ -0,0 +1,3 @@
+.list {
```

Comment:
I wasn't sure if we need a unique name or not here.

Reply:
Nope! The classnames in modules get scoped to their modules.